CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
 * Normalize xcconfig path when generating includes.  
   [bclymer](https://github.com/bclymer)
 
+* Use modern localization identifier 'en' for the development region.
```

Comment:
change log entry was not rebased to latest master it now appears as if it was in 1.8.2 will fix.